build.gradle

```diff
@@ -43,7 +43,11 @@ allprojects {
         options.encoding = 'UTF-8'
         options.incremental = true
     }
-
+    // Jars with the same contents should have the same checksums
+    tasks.withType(AbstractArchiveTask).configureEach {
+        preserveFileTimestamps = false
```

Comment:
If false this ensures that archive entries have the same time for builds between different machines, Java versions and operating systems.